avm/res/desktop-virtualization/scaling-plan/tests/e2e/defaults/main.test.bicep

```
@@ -0,0 +1,30 @@
+targetScope = 'subscription'
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-desktopvirtualization.sp-${serviceShort}-rg'
```

```suggestion
param resourceGroupName string = 'dep-${namePrefix}-desktopvirtualization.scalingplans-${serviceShort}-rg'
```

If the name doesn't happen to get too long (which it shouldn't) I recommend not using abbreviations.